### PR TITLE
Updates to Search topics

### PR DESCRIPTION
**Pull request by diana-lakatos**
Updated the Search topics with more explanations and examples, and added topic on Filtering and Sorting.

### CODE REVIEW

File: marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+## Sorting
+
+Use `sort` to change your the ordering of your collection. 
+When sorting automatically, the `raw` field is used. 
+
+```graphql
+{
+  customizations(
+    page: 1 per_page: 210
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]:
```suggestion
    page: 1,
    per_page: 210
```

File: marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    sort: [
+      { name: "properties.url" order: "ASC" }
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]:
```suggestion
      { name: "properties.url", order: "ASC" }
```

File: marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      { name: "properties.name" order: "DESC" }
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]:
```suggestion
      { name: "properties.name", order: "DESC" }
```

File: marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    email
+  }
+}
+```
+
+
+* `listing` for `listings`
+
+```graphql
+listings(listing: {slug: "transactable-slug"}){
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]:
```suggestion
listings(listing: { slug: "transactable-slug" }){
```

File: marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+Filtering using Elasticsearch is a slightly different operation from [Search](/tutorials/search/search), but they are easy to confuse. This topic explains filtering and sorting using GraphQL queries. 
+
+## Filtering
+
+To filter items by exact value use
+
+* `user:` for `people`
+
+```graphql
+people(user: {first_name: "darek"}) {
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]:
```suggestion
people(user: { first_name: "darek" }) {
```

File: marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+      { name: "properties.url" order: "ASC" }
+      { name: "properties.name" order: "DESC" }
+    ]  	
+    name: "company"
+  ) {
+    total_entries
+    results {
+      id
+      name
+      url property(name: "url")
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]:
```suggestion
      url: property(name: "url")
```

File: marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+      name property(name: "name")
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]:
```suggestion
      name: property(name: "name")
```

File: marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+  }
+}
+``` 
+
+Please note, that there is no filter argument in the customizations entry point. You specify filtering operations directly. 
+
+```graphql
+{
+  customizations(
+    page: 1 per_page: 210
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]:
```suggestion
    page: 1,
    per_page: 210
```

File: marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid (hunk 1 above)
Comment (godot) [reply marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]: @pavelloz no commas are correct and it's correct graphql syntax but i agree for better readability it's OK to change it

Comment (pavelloz) [reply marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid]: im ok with both, but everywhere the same

File: marketplace_builder/views/pages/tutorials/search/search.liquid [hunk 9]
```diff
@@ -7,27 +7,94 @@ slug: tutorials/search/search
 searchable: true
 ---
 
-**Search** functionality in PlatformOS is implemented with GraphQL queries.
+In PlatformOS, we support **searching** and filtering using [Elasticsearch](https://www.elastic.co/learn). 
 
-Search is usually more complex, than other GraphQL queries because of the many additional filters that you can choose from. You can build search for any object supported in GraphQL-most common choices are Users and Transactables.
+Search is usually more complex, than other GraphQL queries because of the many additional filters that you can choose from. You can build search for any object supported in GraphQL. 
+
+## Elastic Search indexes
```
Comment (godot) [inline marketplace_builder/views/pages/tutorials/search/search.liquid]:
```suggestion
## Elasticsearch indexes
```

**Review by godot — APPROVED** [review]

**Pull request merged**